info/container.go

```diff
@@ -405,3 +405,34 @@ func calculateCpuUsage(prev, cur uint64) uint64 {
 	}
 	return cur - prev
 }
+
+type Percentiles struct {
+	// Average over the collected sample.
+	Mean uint64 `json:"mean"`
+	// Max seen over the collected sample.
+	Max uint64 `json:"max"`
+	// 90th percentile over the collected sample.
+	Ninety uint64 `json:"ninety"`
+}
+
+type Usage struct {
+	// Indicates amount of data available [0-100].
+	// If we have data for half a day, we'll still process DayUsage,
+	// but set PercentComplete to 50.
+	PercentComplete int32 `json:"percent_complete"`
+	// Mean, Max, and 90p cpu rate value in milliCpus/seconds. Converted to milliCpus to avoid floats.
+	CpuUsage Percentiles `json:"cpu_usage"`
```

Comment:
I agree to the part of presence checks and it is probably the best approach when we have some passthrough components that just move the data around. For this one, I have two issues:
- It hurts readability quite a bit. No one can figure out what data is being returned just looking at the info file.
- Every component we have in the final summary pipeline actually looks at the data. We'd add up more lookup charges without simplifying much.